CHANGELOG.md

```diff
@@ -1,5 +1,136 @@
+    )}
+  >
+  ```
+
+#### Changes
+
+- <!-- 19 --> [Box] Fix TypeScript error on maxWidth prop (#26859) @ansh-saini
+- <!-- 04 --> [Dialog] Automatically label by its DialogTitle (#26814) @eps1lon
+- <!-- 32 --> [Hidden] Bring back and mark as deprecated (#26908) @siriwatknp
+- <!-- 03 --> [Modal] Type BackdropProps according to styled version (#26836) @eps1lon
+- <!-- 21 --> [Radio] Fix `defaultValue` to match the other value types  (#26945) @oliviertassinari
+- <!-- 48 --> [Stepper] Add completed to global pseudo-classes (#26953) @michal-perlakowski
+- <!-- 25 --> [Stepper] Fix support for no connectors (#26874) @varandasi
+- <!-- 20 --> [TextField] Prevent `hiddenLabel` from spreading to DOM (#26864) @siriwatknp
+- <!-- 18 --> [TextField] Fix label disappearing when focusing a button (#26933) @michal-perlakowski
+
+### `@material-ui/codemod@5.0.0-beta.0`
+
+- <!-- 37 --> [codemod] Add CLI (#26941) @eps1lon
+- <!-- 09 --> [codemod] Correct codemod path (#26932) @BC-M
+- <!-- 23 --> [codemod] Improve DX when using the package (#26936) @oliviertassinari
+
+### @material-ui/icons@5.0.0-beta.0
+
+- <!-- 29 --> [icons] Regenerate transpiled files (#26985) @eps1lon
+
+### @material-ui/lab@5.0.0-beta.0
+
+- <!-- 12 --> [lab] Fix missing dependency on unstyled (#26937) @fishyFrogFace
+
+### `@material-ui/system@5.0.0-beta.0`
+
+- <!-- 14 --> [system] Support array overridesResolver (#26824) @siriwatknp
+
+### Core
+
+- <!-- 46 --> [core] Inline rollup-plugin-size-snapshot (#26986) @eps1lon
+- <!-- 43 --> [core] Remove unused props clone (#26992) @oliviertassinari
+- <!-- 36 --> [core] Fix tests on Windows (#26931) @michaldudak
+- <!-- 31 --> [core] Fix merge conflict between #26954 and #26874 @oliviertassinari
+- <!-- 30 --> [core] Touch dependabot.yml to see if it fixes it @oliviertassinari
+- <!-- 22 --> [core] Upgrade issues-helper to v2 (#26955) @michal-perlakowski
+- <!-- 05 --> [core] Fix merge conflict (#26928) @eps1lon
+- <!-- 45 --> [test] Convert HiddenCSS tests to testing-library (#27019) @eps1lon
+- <!-- 44 --> [test] Convert NativeSelectInput tests to testing-library (#26952) @eps1lon
+- <!-- 39 --> [test] Add a default mount implementation to conformance tests (#26949) @eps1lon
+- <!-- 28 --> [test] Update tests to pass react@next (#26967) @eps1lon
+- <!-- 27 --> [test] Add types to describeConformanceV5 (#26954) @eps1lon
+- <!-- 17 --> [test] Use createPickerMount where appropriate (#26951) @eps1lon
+- <!-- 15 --> [test] Convert SwipeableDrawer tests to testing-library (#26916) @eps1lon
+- <!-- 13 --> [test] Convert Menu tests to testing-library (#26915) @eps1lon
+- <!-- 10 --> [test] Convert Popover tests to testing-library (#26913) @eps1lon
+- <!-- 08 --> [test] Convert Modal tests to testing-library (#26912) @eps1lon
+- <!-- 07 --> [test] Make remaining testing-library tests StrictMode compatible (#26924) @eps1lon
+- <!-- 06 --> [typescript] Disallow spreading TransitionHandlerProps (#26927) @eps1lon
+
+### Docs
```

Comment:
The changes in the docs are more relevant to the developers than the ones on the core infrastructure (core is not the core package). I think that we should move this section before `### Core`.